spark/src/main/java/org/apache/iceberg/actions/RewriteManifestsActionResult.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.iceberg.ManifestFile;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
 
+@Deprecated
```

Comment:
Probably not needed, but you may want to add the same deprecation message here?

Reply:
These are really tightly coupled with the action. I don't see how one could use new action with the old API or just the action result alone. As I am lazy and I was not sure how useful that's going to be, I did not add it.

What do you think, @RussellSpitzer?

Reply:
Yeah I don't really think it's a big deal since I figure no one else is using a RewriteManifestsActionResult